module/VuFind/src/VuFind/View/Helper/Root/MakeLink.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * Make link view helper
+ *
+ * PHP version 7
+ *
+ * Copyright (C) Villanova University 2020.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  View_Helpers
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+namespace VuFind\View\Helper\Root;
+
+use Laminas\View\Helper\AbstractHelper;
+
+/**
+ * Make link view helper
+ *
+ * @category VuFind
+ * @package  View_Helpers
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+class MakeLink extends AbstractHelper
+{
+    /**
+     * Create an anchor tag
+     *
+     * $href will override $attrs['href']
+     * > Feel free to use like makeLink('text', 'href', $defaults);
+     *
+     * If no $href, will try to find an href in $attrs
+     * > makeLink('text', ['href' => '#', 'class' => 'btn-link'])
+     *
+     * If $attrs is a string, will be treated like a class
+     * > makeLink('text', 'btn-link') == makeLink('text', ['class' => 'btn-link'])
+     *
+     * @param string       $text  Link contents
+     * @param string|array $href  Link destination (skippable for (text, attrs))
+     * @param array        $attrs Link attributes (associative array)
+     *
+     * @return string HTML for an anchor tag
+     */
+    public function __invoke($text, $href = null, $attrs = [])
+    {
+        // Skip href
+        if (is_array($href)) {
+            $attrs = $href;
+            $href = null;
+        }
+
+        // $attr string, interpret as class name
+        if (is_string($attrs)) {
+            $attrs = ['class' => $attrs];
+        }
+
+        // Merge all attributes
+        $mergedAttrs = array_merge(
+            [],
+            $attrs ?? [],
+            $href ? ['href' => $href] : []
+        );
+
+        // Plain text on false-y href
+        if (!isset($mergedAttrs['href']) || !$mergedAttrs['href']) {
```

Comment:
Minor point, but `if (!($mergedAttrs['href'] ?? false)) {` might be slightly more concise.